account_invoice_report_ddt_group/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,7 @@
+**Italiano**
+
+Sul partner è possibile impostare "Stampa indirizzo di spedizione nella fattura da DDT".
+
+**English**
+
+In partner form you can set "TD invoice print shipping address".
```

Comment:
L'etichetta della casella di selezione non mi sembra molto chiara.
Dall' `help` vedo che mostra l'indirizzo di spedizione del DDT nella fattura.

In inglese potrebbe risultare più pulita una stringa tipo
`Show (oppure Print) TD shipping address on invoice`

Di conseguenza in italiano verrebbe:
`Mostrare/Stampare indirizzo di spedizione DDT nella fattura`
oppure, come sostantivo:
`Stampa indirizzo di spedizione DDT nella fattura`

Un'alternativa più semplice (e più corta, visto lo spazio ristretto) potrebbe essere:
`TD shipping address on invoice`
`Indirizzo di spedizione DDT nella fattura`